### PR TITLE
Practice Using .map() Function

### PR DESCRIPTION
**Pull request by Tomm1128**
All tests pass. Lab wanted us to practice using the map function. This one was pretty tough and I was only able to solve it after using map twice, in the original function and in the first callback function (removeSpaces).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ const tutorials = [
...
-const titleCased = () => {
-  return tutorials
-}
+const titleCased = () => tutorials.map(removeSpaces)
+
+
+const removeSpaces = (item) => item.split(" ").map(doTitleCase).join(" ")
+
+
+const doTitleCase = (item) => item.charAt(0).toUpperCase() + item.slice(1)
```
Comment (stephenmckeon) [inline index.js:20]:
```suggestion
const titleCased = () => tutorials.map(removeSpaces)

const removeSpaces = (item) => item.split(" ").map(doTitleCase).join(" ")

const doTitleCase = (item) => item.charAt(0).toUpperCase() + item.slice(1)

```

File: index.js [hunk 2]
```diff
@@ -11,6 +11,10 @@ const tutorials = [
   'what is JSONP?'
 ];
 
-const titleCased = () => {
-  return tutorials
-}
+const titleCased = () => tutorials.map(removeSpaces)
```
Comment (stephenmckeon) [inline index.js:14]: `removeSpaces` is misleading. Yes, it does remove spaces, in a way, but it does so much than that, doesn't it?

**Review by stephenmckeon — APPROVED** [review]
Nice job! Love the refactoring.

File: index.js (hunk 2 above)
Comment (Tomm1128) [reply index.js:14]: Yeah your right, it also creates an array where each word is an item in an array. Maybe something like createWordArray?

**Pull request closed without merging**